core/src/main/java/com/linecorp/armeria/server/docs/ServiceSpecification.java

```diff
@@ -149,6 +153,15 @@ public ServiceSpecification(Iterable<ServiceInfo> services,
         this.exampleHeaders = ImmutableList.copyOf(requireNonNull(exampleHeaders, "exampleHeaders"));
     }
 
+    /**
+     * Return a {@link ServiceSpecification} that contains the {@link Route} info of
+     * the attached {@link DocService}.
+     */
+    public ServiceSpecification withDocServiceRoute(Route docServiceRoute) {
+        this.docServiceRoute = docServiceRoute;
+        return this;
+    }
```

Comment:
Can we keep this class immutable? It would also be nice if this method is not public.

Reply:
Got it, I think it makes sense to move this route to `exampleSupport` because it contains path information as well? What do you think, I am not sure about creating a new class just to put a single method 😆

Reply:
That's a good idea! We probably should rename it though. What do you think about `SpecificationSupport`?

Reply:
Never mind. Just realized that you fixed it in a better way.